src/components/home/index.js

```diff
@@ -1,14 +1,13 @@
-import * as clusterActions from 'actions/clusterActions';
```

Comment:
I actually prefer just a good ol dispatch of an action instead of doing the bindActionCreators thing. I think I used `bindActionCreators` when I started because I thought it was needed, but unless anyone disagrees, I'd like to avoid that from now on.

Reply:
see also related change on line 43

Reply:
Never used it, I see this from the docs: 

_The only use case for bindActionCreators is when you want to pass some action creators down to a component that isn't aware of Redux, and you don't want to pass dispatch or the Redux store to it._

And I think it's not the case here, so 👍